lib/dues_calculator.rb

```diff
@@ -23,7 +21,18 @@ def self.dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomina
 
     # The maximum of the two is the total dues per competitor
     [registration_fee_dues, country_band_dues].max
-  rescue Money::Currency::UnknownCurrency, CurrencyUnavailable, Money::Bank::UnknownRate
-    nil
+  end
+
+  def self.error_in_dues_calculation(country_iso2, currency_code)
+    country_band = CountryBand.find_by(iso2: country_iso2)
+    if country_band.nil?
+      "Country band not found."
+    elsif country_band.active_country_band_detail.nil?
+      "Country band detail not found for #{country_band.iso2}."
+    elsif !Money::Currency.table.key?(currency_code.downcase.to_sym)
+      "Currency #{currency_code} is not supported."
+    elsif Money.default_bank.get_rate(currency_code, 'USD').nil?
```

Comment:
@gregorbg This will throw `CurrencyUnavailable` error, and I couldn't find a way to catch that error. So is it fine to have `rescue` within this method and return the error message?